src/player.cpp

```diff
@@ -1506,6 +1506,8 @@ int player::floor_bedding_warmth( const tripoint &pos )
         floor_bedding_warmth -= 500;
     } else if( trap_at_pos.loadid == tr_rollmat ) {
         floor_bedding_warmth -= 1000;
+    } else if( furn_at_pos == f_chair || furn_at_pos == f_bench ) {
+        floor_bedding_warmth -= 1500;
```

Comment:
We should jsonize these values in the future.

Reply:
Would be great, same with other bonuses from this file if possible (`sleepy` and `bonus_fire_warmth` use `furn_at_pos` as well), though I'm not up for this unfortunately. Maybe worth add a `// TODO` until then (there's quite a few them in the file actually)